gorm/filtering_test.go

```diff
@@ -2,11 +2,11 @@ package gorm
 
 import (
 	"context"
+	"github.com/infobloxopen/atlas-app-toolkit/query"
```

Comment:
why the order of imports was changed. Typically we try to break imports in 3 different sections with a newline between each one to separate standard imports, third party imports, imports from infobloxopen

Reply:
Sorry, i think it because i did use standart go format tool and it perform sorting imports as one of format operation, i will try be more careful about this